harness/harness-cli
# feat: [PLG-2612] : Add flags to support service creation for GCP flow

Tests:
1. Create service for GCP flow without providing the `gcp-project-name` and `gcp-bucket-name`. 
Expected Behavior: CLI prompts the user for above flags on the fly.
![image](https://github.com/harness/harness-cli/assets/117125370/c8462f9a-8589-475b-9d0d-70c7fad99b71)
2. Create service for GCP flow providing the `gcp-project-name` and `gcp-bucket-name`. 
Expected Behavior: no  prompts required for the user and goes ahead and creates the service without any interruption. 
![image](https://github.com/harness/harness-cli/assets/117125370/af2163db-8741-4f70-80d4-169051e971b6)
